src/main/java/console/AuctionMenuManager.java

```diff
@@ -90,7 +92,16 @@ private void closeAuction() {
         auctionManager.closeAuction(auctionId, currentUser);
         Auction closedAuction = auctionManager.getAuctions().get(auctionId);
         AuctionSummary auctionSummary = closedAuction.getAuctionSummary();
-        out.println(auctionSummary.summarize());
+        out.println("Auction ID: " + auctionSummary.auctionId());
```

Comment:
I'd consider separating the functions of performing actions from the functions of printing output. It gives more flexibility in terms of swapping out the rendering in the future. Create an interface AuctionInformationDisplay with methods like displayAuctionInfo, displayAuctionSummary, etc. Then an implementation that prints these things to the console. A different implementation could show them in a UI. 

so closeAuction would be like auction closing the auction and getting the AucitonSummary then auctionInformationDisplay.displayAuctionSummary(auctionSummary)